jobs/check-aws-regions.Jenkinsfile

```diff
@@ -1,3 +1,5 @@
+/* Note: AWS region enablement, refer to the documentation here: [AWS Region Enablement](docs/aws-region-enable.md) */
```

Comment:
I feel like maybe we should include the link (i.e. clickable) in the slack message that gets sent to the channel.

i.e. the notification sends us somewhere with the steps to take. we don't have to remember where the docs are.